rust/relay/README.md

```diff
@@ -16,28 +16,48 @@ Relaying of data through other means such as DATA frames is not supported.
 
 ## Building
 
-You can build the server using: `cargo build --release --bin relay`
+You can build the relay using: `cargo build --release --bin firezone-relay`
+
+You should then find a binary in `target/release/firezone-relay`.
 
 ## Running
 
-For an up-to-date documentation on the available configurations options and a detailed help text, run `cargo run --bin relay -- --help`.
-All command-line options can be overridden using environment variables.
-Those variables are listed in the `--help` output at the bottom of each command.
```

Comment:
It would be good to still refer users to `--help`.

Reply:
Ah, yeah

Reply:
It's still here, just moved down a bit.